package.json

```diff
@@ -16,24 +17,24 @@
   "author": "Marshall Polaris <marshall@pol.rs>",
   "license": "MPL-2.0",
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=22.0.0"
```

Comment:
This probably ought to match line 108

Reply:
Good catch.  Thanks.

This change is from PR #145 so I've applied it there.